src/components/Thread.js

```diff
@@ -45,8 +45,8 @@ export const Thread = withRouter((props) => {
           <h3 className="text-center">
           <button
             type="button"
-            className={"btn btn-link btn-lg"}
-            onClick={() => {props.history.push('/community/' + data.thread.group.id)}}>
+            className="btn btn-link btn-lg"
+            onClick={() => {props.history.push(`/community/${  data.thread.group.id}`)}}>
```

Comment:
Same suggestion here in terms of readability of the string:
```suggestion
            onClick={() => {props.history.push(`/community/${data.thread.group.id}`)}}>
```

Reply:
We will we using an internal `<Button />` component for this in the future